packages/phone-number-privacy/common/src/utils/authentication.ts

```diff
@@ -108,33 +111,37 @@ export async function isVerified(
   contractKit: ContractKit,
   logger: Logger
 ): Promise<boolean> {
-  return retryAsyncWithBackOffAndTimeout(
```

Comment:
I believe you can also catch the error like:
`return retryAsyncWithBackOffAndTimeout(....).catch(error => { ... })`

Reply:
Yea you should be able to. The reason I changed it to use the try/catch syntax is for better type checking on the error. When I switched to this syntax I uncovered what I think is the cause of the issue, so I feel like this syntax may be better for avoiding this type of bug.